kolibri/core/content/models.py

```diff
@@ -447,7 +447,6 @@ def save(self, *args, **kwargs):
         """
         Save override to set type for the proxy models
         """
-        self.type = getattr(self.__class__.objects, "request_type", None)
         return super(ContentRequest, self).save(*args, **kwargs)
```

Comment:
Looking at this again - I am wondering if this is still useful to have, just in case we do any saves outside of the context of build_for_user?

Unless there was a reason this needed to be deleted - in which case we should remove the entire method, as it's just making a super call and nothing more.

Reply:
Would like follow up on this comment to resolve it.